build/apis/githubapi-types.d.ts

```diff
@@ -78,16 +65,30 @@ export interface StatusEvent {
 
 // Github API -----------------------------------------------------------------
 // These provide the current bare minimum definitions for child Procbots working with them.
+export interface Comment {
+    body: string;
+    user: {
+        login: string;
+        type: string;
+    };
+    created_at: string;
+    updated_at: string;
```

Comment:
Why are we not using `Date` objects to store this? As this is going to be a library, seems like that would make the most sense. Gonna leave it up to you though obviously.